packages/web-components/fast-foundation/src/progress/progress.template.ts

```diff
@@ -27,7 +27,7 @@ export const ProgressTemplate: ViewTemplate<BaseProgress> = html`
             `
         )}
         ${when(
-            x => !x.value,
+            x => typeof x.value !== "number" || Number.isNaN(x.value),
```

Comment:
Shouldn't the `nullableNumberConverter` convert value to `null` if the value is `null`, `undefined` or `NaN`? Therefore should you be checking for `null` value not `NaN`?

https://github.com/microsoft/fast/blob/master/packages/web-components/fast-element/src/components/attributes.ts#L87

Reply:
Good catch, @eljefe223 sorry for the miss-guidance on this.  A simple null check should be fine, but also update the TS type for the property to be `number | null` so the property type matches the converter's return type.